modulesAndFiles/tests.c

```diff
@@ -0,0 +1,50 @@
+#include "tests.h"
+#include "sortingAndSearching.h"
+#include "readingFromFile.h"
+#include <stdio.h>
+#include <stdbool.h>
+
+bool runTests(int array[], int size, int expectedResult) {
+    int result = findMostFrequent(array, size);
+    if (result == expectedResult) {
+        return true;
+    } else {
+        return false;
+	}
```

Comment:
```suggestion
    return result == expectedResult;
```
Тут у Вас и с отступами жесть какая-то